library/dokku_ps_scale.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.dokku_utils import subprocess_check_output
+import subprocess
+import re
+
+DOCUMENTATION = """
+---
+module: dokku_ps_scale
+short_description: Manage process scaling for a given dokku application
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  scale:
+    description:
+      - A map of scale values where proctype => qty
+    required: True
+    default: {}
+    aliases: []
+  skip_deploy:
```

Comment:
as for the naming, I guess starting from scratch the canonical name would be `deploy` (which I would slightly prefer) but since you are mirroring the language from the dokku cli `skip_deploy` is also valid (no need to change).

Reply:
Thanks - this was very much taken from the naming of the method on `ps:scale` itself - I think I'd be most comfortable leaving the naming consistent with that. I feel this will be a relatively infrequently used option.